deploy/clowdapp.yaml

```diff
@@ -265,12 +259,4 @@ parameters:
   - description: Internal queue type (memory/sqs/postgres).
     name: WORKER_QUEUE
     value: "redis"
-  - description: Internal queue concurrency, how many goroutines to run in parallel. Not used in 'redis' mode.
-    name: WORKER_CONCURRENCY
-    value: "100"
-  - description: Internal queue heartbeat interval. How often to update job liveness information. Not used in 'redis' mode.
-    name: WORKER_HEARTBEAT
-    value: "30s"
-  - description: Internal queue beat threshold. Now many times the beat can be updated until job is cancelled. Not used in 'redis' mode.
-    name: WORKER_MAX_BEATS
-    value: "10"
+
```

Comment:
All of these settings were completely removed, I replaced them with:

* Poll interval - how often to poll Redis for new messages (essentially timeout).
* Maximum number of goroutines per worker instance - by default it is CPUs + 1 when unset.

These are now in app configuration, tho, I am not adding them in clowder until we need them. I think defaults are okay.